weltyt.py

```diff
@@ -109,6 +109,13 @@ def buy_lands(lands):
             cur.execute(
                 f'UPDATE resources SET amount = {new_money_amount}  WHERE type = "Money";'
             )
+            last_id_number = cur.execute(
```

Comment:
You don't need to write your own logic for that.
See: https://www.tutorialspoint.com/sqlite/sqlite_using_autoincrement.htm and `PRIMARY KEY`.